pkg/asset/installconfig/aws/validation.go

```diff
@@ -493,7 +494,12 @@ func validateSecurityGroupIDs(ctx context.Context, meta *Metadata, fldPath *fiel
 		return append(allErrs, field.Invalid(fldPath, vpc, errMsg))
 	}
 
-	securityGroups, err := DescribeSecurityGroups(ctx, meta.session, pool.AdditionalSecurityGroupIDs, platform.Region)
+	session, err := meta.Session(ctx)
+	if err != nil {
+		return append(allErrs, field.InternalError(fldPath, fmt.Errorf("unable to start a session: %w", err)))
```

Comment:
nit: technically this may not be "starting" a session. Maybe change to something like "unable to retrieve aws session: %w" (or something similar).